packages/pluggableWidgets/slider-web/src/utils/createHandleGenerator.tsx

```diff
@@ -42,10 +43,11 @@ export function createHandleGenerator(props: CreateHandleGeneratorParams): Handl
 
         return (
             <Tooltip
+                defaultVisible
                 prefixCls="rc-slider-tooltip"
                 overlay={isCustomText ? overlay : restProps.value}
                 trigger={["hover", "click", "focus"]}
-                visible={dragging}
+                visible={tooltipAlwaysVisible || dragging}
```

Comment:
Nit: you can also do `tooltipAlwaysVisible ?? dragging`, then you don't need the default value 😄

Reply:
@Keraito, I'm not sure your nit is actually correct.
1. `tooltipAlwaysVisible` will be always boolean and it will be `false` by default because of `defaultValue` attribute in `Slider.xml`
2. Since this var always has boolean type and never will be `undefined` or `null` using `??` we will never reach `dragging` value
So, in my view we should leave `||` operator as it's intended behavior.

Reply:
That’s why I mentioned you don’t need the default value. It’s either || + default value or ?? + no default value. Because right now you define it as optional in props typing, but you add the default value when destructuring. So in typing it’s boolean | undefined | null, but in code it’s boolean. This is different and can be confusing. If possible, I like to avoid default values. But as mentioned, it’s a nit and I’m fine with keeping it as is.

Reply:
@Keraito, fixed